--- python-edje-0.7.3/edje.orig/edje.c_edje_object.pxi	2010-10-01 01:03:42.000000000 +0200
+++ python-edje-0.7.3/edje/edje.c_edje_object.pxi	2011-12-17 10:08:38.912385405 +0100
@@ -139,7 +139,7 @@
         ...
         >>> my_edje.message_handler_set(msg_dbg)
     """
-    def __new__(self, *a, **ka):
+    def __cinit__(self, *a, **ka):
         self._signal_callbacks = {}
 
     def __init__(self, c_evas.Canvas canvas not None, **kargs):
